src/bin/delete-crate.rs

```diff
@@ -18,6 +18,7 @@ use std::io::prelude::*;
 
 use cargo_registry::Crate;
 
+#[allow(dead_code)]
 fn main() {
```

Comment:
iirc the "proper" way to do this is `#[cfg(not(test))]`. May be misremembering for `doc`.